README.md

```diff
@@ -75,16 +75,31 @@ GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 f
    cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo && make
    ```
 
-4. Install OpenSpades (optional but recommended):
+4. Launching the game (follow the steps either A or B)
 
-   `sudo make install`
+   **4.A.1. Install OpenSpades (recommended):**
```

Comment:
Perhaps we can restructure this to better suit the Markdown style and reduce the nesting depth. How about splitting out the installation part into its own section? The final structure would look like this:

> ### On Linux, from source
>
> #### Building
>
> 1. Install dependencies:
>
> [...]
>
> 3. . Create directory `openspades.mk` in cloned/downloaded openspades repo and compile: [...]
>
> #### Installing and launching
>
> To launch the built game without installing:
>
> ```
> cd $REPO_DIRECTORY/openspades.mk; bin/openspades
> ```
>
> To install the game to your system (recommended), take the following steps:
>
> 1. [...]
> 2. Launch the game by typing `openspades` into command line
>
> Alternatively, to install the game to a different directory, take the following steps:
>
> 1. [...]
> 2. [...]
>
> **note**: If you have a previous installation of OpenSpades, you have to uninstall it manually by `sudo rm -rf /usr/local/share/games/openspades` before installing a new one, or else it might load old resources.